docs/spec/qualifiers.rst

```diff
@@ -213,35 +213,44 @@ following should be allowed::
 Syntax
 ^^^^^^
 
-``Annotated`` is parameterized with a type and an arbitrary list of
-Python values that represent the annotations. Here are the specific
-details of the syntax:
+``Annotated`` is parameterized with a *base type* and an arbitrary list of
+Python values that represent *metadata* for the type::
 
-* The first argument to ``Annotated`` must be a valid type
+    from typing import Annotated
 
-* Multiple type annotations are supported (``Annotated`` supports variadic
+    Annotated[BaseType, Metadata1, Metadata2, ...]
+
+Here are the specific details of the syntax:
+
+* The base type (the first argument to ``Annotated``) must be a valid type
+
+    * If ``Annotated`` is used in a place where arbitrary
+      :term:`annotation expressions <annotation expression>` are allowed,
+      the base type may also be an annotation expression
+    * Otherwise, the base type must be a valid :term:`type expression`.
+
+* Multiple metadata objects are supported (``Annotated`` supports variadic
   arguments)::
 
     Annotated[int, ValueRange(3, 10), ctype("char")]
 
-* ``Annotated`` must be called with at least two arguments (
-  ``Annotated[int]`` is not valid)
+* There must be at least one metadata object (``Annotated[int]`` is not valid)
 
-* The order of the annotations is preserved and matters for equality
+* The order of the metadata is preserved and matters for equality
```

Comment:
Could refer to these as metadata "elements" rather than "objects"

```suggestion
* The order of the metadata elements is preserved and matters for equality
```

Reply:
I think in this line it's fine to just say "The order of the metadata".

Reply:
I don't mind this line too much, but I was intending to also make a broader point -- you use "metadata objects" quite a bit; I personally prefer "metadata elements" as a term.

Reply:
Edited to say "elements"